ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -135,3 +135,23 @@ Verify Pipeline Run Deployment Is Successful    # robocop: disable
     ${containerStatuses}=  Create List        terminated    terminated
     ...    terminated    terminated    terminated
     Verify Deployment    ${valid_model}  1  1  ${containerNames}    ${podStatuses}    ${containerStatuses}
+
+Delete Pipeline Ran
+    [Documentation]    Delete a pipeline that ran based on name and pipeline. From the left menu select
+    ...                "Data Science Pipelines" -> Runs. In the new page, select the tab "Triggered".
+    ...                The "Delete Pipeline Ran" will search for a line in the grid that match the pipeline name and
+    ...                the run name. Based on that, hit the ... Menu in the row and hit Delete drop down menu.
+    [Arguments]    ${run_name}
+    Click Element    //button[@id="pipelines"]
+    Click Element    //a[text()='Runs']
+    Wait Until Page Contains Element    xpath://span[text()='Triggered']
+    Click Element    //span[text()='Triggered']
+    Wait Until Page Contains Element    xpath://table//tr[td[div[p[text()='${run_name}']]]]
+    Wait Until Page Contains Element    //table//tr[td[div[p[text()='${run_name}']]]]//td[contains(@class, 'pf-c-table__action')]/div
+    Click Element    //table//tr[td[div[p[text()='${run_name}']]]]//td[contains(@class, 'pf-c-table__action')]/div
+    Click Element    //table//tr[td[div[p[text()='${run_name}']]]]//td[contains(@class, 'pf-c-table__action')]/div/ul/li/button[contains(text(), 'Delete')]
```

Comment:
could you try either using this kw `Click Action From Actions Menu`  (`ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource`) or create a similar one for the Run?

Reply:
it might be useful to reduce the amount of duplicated code in other tests/kw about runs

Reply:
Done. The code inside `Pipelines.resource` changed. The current one wasn't working

Reply:
None at the moment. It should be created